### DISCUSSION TITLE
since it's not a bug, I'll write here)

### DISCUSSION THREAD

**Discussion by ops244** [make-all/tuya-local#1508] [Q&A]
is it possible to add a "return to base" button to this vacuum cleaner? I tried, but it didn't work out.

https://github.com/make-all/tuya-local/issues/1284

![2024-01-11_08-47-29](https://github.com/make-all/tuya-local/assets/19797600/1ec9c3b2-ca8c-4c84-aeb4-46a298dae8f9)

https://github.com/make-all/tuya-local/issues/1284

**Comment by make-all** [comment]
This is actually a bug, so I created #1509 for it
